tests/unit/linking/test_link.py

```diff
@@ -68,7 +63,18 @@ def test_compare_match(self):
             }
         )
 
-        algorithm_pass = models.AlgorithmPass(id=1, algorithm_id=1, blocking_keys=[1], evaluators={"first_name": "func:recordlinker.linking.matchers.feature_match_exact", "last_name": "func:recordlinker.linking.matchers.feature_match_fuzzy_string"}, rule="func:recordlinker.linking.matchers.eval_perfect_match", cluster_ratio=1.0, kwargs={})
+        algorithm_pass = models.AlgorithmPass(
+            id=1,
+            algorithm_id=1,
```

Comment:
What is the difference between the `id` and the `algorithm_id`?

Reply:
`id` is the primary key on the algorithm_pass table.  `algorithm_id` is the primary key on the algorithm table.